mcs/class/Makefile

```diff
@@ -50,6 +50,7 @@ endif
 
 mobile_common_dirs := \
 	corlib	\
+	Mono.Profiler.Log	\
```

Comment:
The MLPD reading is not the important part for mobile. The `LogRuntimeProfiler` class, when actually implemented, will contain methods for interacting with the profiler at runtime. The code isn't there yet, but I don't see any harm in including the library for mobile now to make sure it actually builds, especially since changes need to be made in XA/XI for it to be included there anyway, which we can do when the library is ready.